code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -264,6 +265,10 @@
 
 	var/need_round_end_check = FALSE
 
+	//Joining as Facehugger vars
+	var/hugger_timelock = 15 MINUTES //You can only join as a hugger 15 minutes into the round
+	var/playable_hugger_limit = 0 //How many huggers can the hive support
```

Comment:
```suggestion
	/// When can huggers join the round
	var/hugger_timelock = 15 MINUTES
	 /// How many huggers can the hive support
	var/playable_hugger_limit = 0
```
i think walter meant to dm doc these vars - so they show up on mouseover in vsc, but this is optional regardless